packages/phone-number-privacy/src/common/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Response } from 'firebase-functions'
```

Comment:
This file feels a bit like a bunch of random stuff.
Consider breaking out ErrorMessages into its own file